packages/tests/server/websockets.test.ts

```diff
@@ -48,6 +56,31 @@ function factory(config?: {
       await waitMs(50);
       return 'slow query resolved';
     }),
+    iterable: t.procedure
+      .input(
+        z
+          .object({
+            lastEventId: z.coerce.number().nullish(),
+          })
+          .nullish(),
+      )
+      .subscription(async function* (opts) {
+        let from = opts?.input?.lastEventId ?? 0;
+
+        while (true) {
+          from++;
+          await iterableDeferred.promise;
+          const msg: Message = {
+            id: String(from),
+            title: 'hello ' + from,
+          };
+
+          yield sse({
```

Comment:
Again, mixing in sse into websockets feels a bit off. Seems like there's some common parts here that can use a unified name?